lib/server/loop.js

```diff
@@ -52,12 +52,16 @@ function init (env, ctx) {
         , keyId: env.extendedSettings.loop.apnsKeyId
         , teamId: env.extendedSettings.loop.developerTeamId
       },
-      production: false
+      production: env.extendedSettings.loop.pushServerEnvironment === "production"
```

Comment:
please be lenient with case errors and use a `toLowerCase()`, similar to https://github.com/nightscout/cgm-remote-monitor/blob/f81298fb287e4a93c4e05ef23bca5b2ac2bf6b78/env.js#L26